docs/data/product/dea-intertidal/_quality.md

```diff
@@ -22,7 +20,10 @@ To create a consistent basis for comparison against DEA Intertidal outputs, data
 | MAE (m)     | 0.21       | 0.21      | **0.20**   |
 | Bias (m)    | 0.14       | 0.16      | **0.12**   |
 | Slope       | 0.44       | 0.80      | **1.03**   |
-:::
+
+<figure>
```

Comment:
Hey @benji-glitsos-ga , I noticed in the latest preview the Table figure caption is below the table instead of above:

![image](https://github.com/GeoscienceAustralia/dea-knowledge-hub/assets/17680388/d0afa85b-a181-428b-bf56-1ccd97e5e388)

Reply:
@robbibt I wasn't aware until now that table captions must be above. I'll change it back